client/blocks/site-preview/index.jsx

```diff
@@ -1,35 +1,119 @@
 /** @format */
-
 /**
  * External dependencies
  */
-
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import debugFactory from 'debug';
 
 /**
  * Internal dependencies
  */
-import UrlPreview from 'blocks/url-preview';
+import { closePreview } from 'state/ui/preview/actions';
+import { getPreviewSite, getPreviewSiteId, getPreviewUrl } from 'state/ui/preview/selectors';
+import { getSiteOption, getSiteSlug } from 'state/sites/selectors';
 import { getCurrentLayoutFocus } from 'state/ui/layout-focus/selectors';
+import { addQueryArgs } from 'lib/route';
+import isDomainOnlySite from 'state/selectors/is-domain-only-site';
+import WebPreview from 'components/web-preview';
+
+const debug = debugFactory( 'calypso:design-preview' );
 
-class SitePreview extends Component {
+class UrlPreview extends Component {
```

Comment:
should this be SitePreview since it's in `site-preview`?

Reply:
Good point. Renamed `UrlPreview` to `SitePreview`.